src/libraries/System.Text.Json/tests/Common/CollectionTests/CollectionTests.AsyncEnumerable.cs

```diff
@@ -18,18 +18,23 @@ public abstract partial class CollectionTests
         [MemberData(nameof(GetAsyncEnumerableSources))]
         public async Task WriteRootLevelAsyncEnumerable<TElement>(IEnumerable<TElement> source, int delayInterval, int bufferSize)
         {
+            if (StreamingSerializer is null)
+            {
+                return;
+            }
+
             JsonSerializerOptions options = new JsonSerializerOptions
             {
                 DefaultBufferSize = bufferSize
             };
 
-            string expectedJson = await JsonSerializerWrapperForString.SerializeWrapper(source);
+            string expectedJson = JsonSerializer.Serialize(source);
 
             using var stream = new Utf8MemoryStream();
             var asyncEnumerable = new MockedAsyncEnumerable<TElement>(source, delayInterval);
-            await JsonSerializerWrapperForStream.SerializeWrapper(stream, asyncEnumerable, options);
+            await StreamingSerializer.Serialize(stream, asyncEnumerable, options);
```

Comment:
Past discussion around naming was that we didn't want to use the same method name "Serialize" and "Deserialize" as the `JsonSerializer` to make code searches easier. I don't have a strong opinion either way, but that's the reasoning behind the current names.